conn.go

```diff
@@ -117,6 +117,11 @@ type conn struct {
 	// debugging.
 	disablePreparedBinaryResult bool
 
+	// If set, this connection will not skip returning results for
+	// multi-statement queries where one of the statements does not return
+	// results such as CREATE TABLE or SET.
+	disableSkipZeroResultQueries bool
```

Comment:
The double negative of `disableSkip` is hard to parse. How about `enableMultipleResults`, since that's what you're really opting in to here?